Libraries/LibWeb/CSS/StyleValues/TransformationStyleValue.cpp

```diff
@@ -23,14 +23,79 @@
 #include <LibWeb/CSS/PropertyID.h>
 #include <LibWeb/CSS/Serialize.h>
 #include <LibWeb/CSS/StyleValues/AngleStyleValue.h>
+#include <LibWeb/CSS/StyleValues/KeywordStyleValue.h>
 #include <LibWeb/CSS/StyleValues/LengthStyleValue.h>
 #include <LibWeb/CSS/StyleValues/NumberStyleValue.h>
 #include <LibWeb/CSS/StyleValues/PercentageStyleValue.h>
+#include <LibWeb/CSS/StyleValues/TransformationStyleValue.h>
 #include <LibWeb/CSS/Transformation.h>
 #include <LibWeb/Geometry/DOMMatrix.h>
 
 namespace Web::CSS {
 
+ValueComparingNonnullRefPtr<TransformationStyleValue const> TransformationStyleValue::identity_transformation(
```

Comment:
This could maybe be generated too, but that might be more code than this is. :sweat_smile: